src/Utils/Hexagons.svelte

```diff
@@ -148,64 +171,178 @@ $: drawableHexagons = hexagons.map(({id, row, col}) => {
     ];
     const x = (globalPadding / 2) + (col/2 + .5) * width + Math.max((col / 2) * horizontalSpacing, 0);
     const y = (globalPadding / 2) + ((row/2) * 1.5 + .5) * height + Math.max((row / 2) * verticalSpacing, 0);
-    const color = activeHexagon === id ? "mediumseagreen" : "black";
+    const color = activeHexagon.includes(id) ? "mediumseagreen" : "black";
 
     return {id, x, y, width, height, points: points.join(' '), color};
 });
 
+function sendCommandBlocks() {
+    (async () => {
+        return await Com.hitEndpoint(endpoint, nopRoute, $act_command);
+    })().then(result => {
+        if (result.hasOwnProperty('Failure')) {
+            $message = result['Failure']['message']; 
+        } 
+        return result;
+    }).catch(error => {
+        $message = error;
+        activeHexagon = [];
+        throw error;
+    });
+}
+
+function findActiveHexagons(e) {
+    const radius = Math.sqrt(3) * hexagonSideLength / 2;
+    const radiusSquared = radius * radius;
+    activeHexagon = [];
+    let Xpos;
+    let Ypos;
+    for(var i = 0; i < drawableHexagons.length; i++) {
+        const x = drawableHexagons[i].x;
+        const y = drawableHexagons[i].y;
+        for (var j = 0; j < numTouches; j++) {
+            if (isTouch) {
+                Xpos = e.targetTouches[j].clientX - boundRect.left;
+                Ypos = e.targetTouches[j].clientY - boundRect.top;
+            } else{
+                Xpos = e.offsetX;
+                Ypos = e.offsetY;
+            }
+            const xDist = (x - Xpos);
+            const yDist = (y - Ypos);
+            const euclidianDistSquared = xDist * xDist + yDist * yDist;
+            if(euclidianDistSquared < radiusSquared) {
+                activeHexagon = [...activeHexagon, drawableHexagons[i].id];
+                buildCommandBlocks(activeHexagon);
+            }
+        }
+    }
+    sendCommandBlocks();
+}
+
 function handleTouchStart(e) { 
     if(e.stopPropagation) e.stopPropagation();
     if(e.preventDefault) e.preventDefault();
     mouseDown = true;
+    clearInterval(resendCommand);
+    clearTimeout(pendingTimeout);
+    try {
+        if (e.targetTouches.length > 0) {
+            isTouch = true;
+            numTouches = e.targetTouches.length;
+        }
+    } catch(error) { 
+        numTouches = 1;
+    }
+
+    if (isTouch) {
+        Xstart = e.targetTouches[0].clientX - boundRect.left;
+        Ystart = e.targetTouches[0].clientY - boundRect.top;
+    } else {
+        Xstart = e.offsetX;
+        Ystart = e.offsetY;
+    }
+
+    if(!isPreset) {
+        findActiveHexagons(e);
+        resendCommand = setInterval(findActiveHexagons, 500, e);
```

Comment:
seems like this interval will be easy enough to make configurable on the presets page